drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -664,6 +650,21 @@ int stm32_clock_control_init(const struct device *dev)
 	new_flash_freq = RCC_CALC_FLASH_FREQ(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC,
 				      STM32_FLASH_PRESCALER);
 
+#if defined(STM32_PLL_ENABLED) && defined(CONFIG_SOC_SERIES_STM32F7X)
```

Comment:
LGTM, but in order to ease future maintenance, I would have move this block (STM32_PLL_ENABLED && CONFIG_SOC_SERIES_STM32F7X) before FLASH_ACR_LATENCY block (to avoid imbrication of both compilation switches)

Reply:
Yes, 
activation of the overdrive depends on the sys clock freq (whatever the latency but must be done before the latency setting)